webclient/app/src/app/pages/appTemplate/AppTemplateOverview.jsx

```diff
@@ -35,6 +38,13 @@ function AppTemplateOverview() {
         });
     };
 
+    useEffect(() => {
+        const userRest = new UserRest();
```

Comment:
I would prefer to create one instance of UserRest() on the top with the use of useMemo

Reply:
done also for GeneralSection.jsx